### DISCUSSION TITLE
dxf file for top plate

### DISCUSSION THREAD

**Discussion by crembz** [duckyb/eternal-keypad#6] [Ideas]
Hi there, 

Looking to get a brass top plate cut and need to provide a dxf file. I found the one for the bottom plate but am stuggling to understand how I can create one out of kicad for the top plate. My results converting the gerber files dont produce the drill holes properly. Can you please generate this one file?

**Comment by duckyb** [comment]
Hi @crembz, I have added the file you requested in the [release assets](https://github.com/duckyb/eternal-keypad/releases/tag/v1.2).

**Reply by crembz** [reply]
Perfect thank you so much for that!
